hooks/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+
+if [ "$branch" = "main" ]; then
+  echo "You can't commit directly to main branch."
```

Comment:
I think the message should be more clear.

Reply:
I think the message is clear enough.